Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -1371,6 +1367,8 @@ public extension Purchases {
         )
     }
 
+    #if !ENABLE_CUSTOM_ENTITLEMENT_COMPUTATION
+
```

Comment:
Just to confirm that we really want to remove this `configure(with builder: Configuration.Builder)` when `ENABLE_CUSTOM_ENTITLEMENT_COMPUTATION`

Reply:
TBH, this feels a bit wrong to me, considering we shouldn't be passing the builder object directly, unless we wanted to do additional changes, which we shouldn't... So, since I believe there was no need I hid it. Lmk if you think otherwise!

Reply:
Yeah, I think it makes sense. Thanks for the explanation!